data/core/init.lua

```diff
@@ -345,10 +354,6 @@ function core.add_project_directory(path)
     shown_subdir = {},
   }
   table.insert(core.project_directories, dir)
-  scan_project_folder(#core.project_directories)
```

Comment:
I thought that this was the best place to have it, why was this moved away?

Reply:
Because it needs to occur after project and user plugin load.

Reply:
So this line does need to be here, you're absolutely right. But, it also has to happen after project module load. Maybe we need to add a boolean to the function to determine whether or not to scan? I'll try and work something up.

Reply:
OK; I've added that in.